eng/Versions.props

```diff
@@ -1,12 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <MajorVersion>8</MajorVersion>
+    <MinorVersion>0</MinorVersion>
     <VersionPrefix>19.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
+    <PreReleaseVersionIteration>4</PreReleaseVersionIteration>
+    <WasiVersion>19.0.0</WasiVersion>
```

Comment:
We should either use VersionPrefix in lieu of WasiVersion, or make `<WasiVersion>$(VersionPrefix)</WasiVersion>`. No point having the same info twice